ibtk/src/lagrangian/FEDataManager.cpp

```diff
@@ -632,9 +667,9 @@ FEDataManager::reinitElementMappings()
                 "At least one node in the current mesh is inside the fluid domain and not associated with any "
                 "patch. This class currently assumes that all elements are on the finest level and will not "
                 "work correctly if this assumption does not hold. This usually happens when you use multiple "
-                "patch levels and set the regrid CFL interval to a value larger than one. To change this check set "
-                "node_outside_patch_check to a different value in the input database: see the documentation of "
-                "FEDataManager for more information.";
+                "patch levels and set the regrid CFL interval to a value larger than one. To change this check "
+                "set node_outside_patch_check to a different value in the input database: see the documentation "
+                "of FEDataManager for more information.";
```

Comment:
I'm not sure why clang-format started complaining about this.

Reply:
🤷‍♂️